Makefile

```diff
@@ -56,3 +56,7 @@ ifeq (${OSFLAG},linux)
 else
 	@$(MAKE) build-darwin
 endif
+
+.PHONY: build-image
+build-image: ## Build the container image
+	docker build -t ${BINARY}:latest .
```

Comment:
I wrestled with `latest` here, vs `devel` or adding in a make task for using the actual tag.
After a minute or two, I think `latest` is fine.